ci/owncloud/upload_to_owncloud.py

```diff
@@ -1,8 +1,46 @@
 from os import path
 from getpass import getuser
+import netrc
 from webdav3.client import Client
+import json
 from opts import debug, message, opts
 
+default_hostname = "https://ftp.essensium.com/owncloud/remote.php/dav/files/"
+
+
+def get_credentials(options_path=None, force_netrc=True):
```

Comment:
As I said, this is redundant. There is absolutely no gain from having 2 different files for fetching credentials.
What you can do is add an optional param `netrc_path` which will default to `~/.netrc`.

Reply:
Not exactly redundant as the `.netrc` file's hostanme field contains only a partial path, we could store only the `"owncloud/remote.php/dav/files/"` part, but in the shell version it's stored the way it was implemented here.